### PR TITLE
Extract Text from Images Using Python Script

### PR DESCRIPTION
**Pull request by avinashkranjan**
<h4> OCR-Image-to-Text </h4>
<summary> It's a simple script to extract text from any image.  </summary>

Example:
Image Used:

img

Output:

Screenshot (124)

### CODE REVIEW

**Comment by avinashkranjan** [comment]
#93 OCR-Image-to-Text
It's a simple script to extract text from any image.

File: Image-Processing/ocr-img-to-txt.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+from PIL import Image
+
+
+# path to the Tessaract
+pytesseract.pytesseract.tesseract_cmd = r"C:\Program Files\Tesseract-OCR\tesseract.exe"
+
+
+def convert():
+    img = Image.open(
+        r"E:\Python-Programs\PythonProjects\OCR-Image-To-Text\Img-To-Convt\img1.jpg")
```
Comment (disha2sinha) [inline Image-Processing/ocr-img-to-txt.py:11]: bring this in same line with line 10

File: Image-Processing/ocr-img-to-txt.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    text = pytesseract.image_to_string(img)
+    print(text)
+
+
```
Comment (disha2sinha) [inline Image-Processing/ocr-img-to-txt.py:15]: remove this new line

File: Image-Processing/ocr-img-to-txt.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import pytesseract
+from PIL import Image
+
+
```
Comment (disha2sinha) [inline Image-Processing/ocr-img-to-txt.py:4]: remove this new line

**Review by disha2sinha — CHANGES_REQUESTED** [review]

**Comment by disha2sinha** [comment]
Thank you for your contribution @avinashkranjan 
Before I approve changes to the PR, I will request you the following changes mentioned below.

    Add README (guide) along with this inside the folder same as your <filename>
    Update the README with appropriate installation methods and description guidelines.
    Maintain proper Contribution protocols (ignore if you're already following it!) & good practices!

**Comment by avinashkranjan** [comment]
#155 
@disha2sinha I have made the necessary Changes moved it to a new folder.. Have a look and give the review..
[Here's the Link to New Pull Request](https://github.com/ankitdobhal/Awesome-Python-Scripts/pull/155)

**Pull request closed without merging**